configs/defis/config.yaml

```diff
@@ -0,0 +1,129 @@
+# config file for meteo-france
+
+datagouvfr:
+  # data.gouv.fr base URL
+  base_url: https://www.data.gouv.fr
+  # oauth settings
+  oauth_client_id:
+  # pkce client secret, explicitely public
+  oauth_client_secret:
+
+universe:
+  # "universe" topic id
+  topic_id: 6571f222129681e83de11aa2 # meteo-test
+  # universe name, used as bouquets tag
+  name: defis
+
+# UI customizations
+website:
+  title: 'defis.data.gouv.fr'
+  rf_title: 'RÉPUBLIQUE<br>
+    FRANÇAISE'
+  service_logo: '/logos/defis-logo.svg'
+  badge:
+    display: true
+    text: 'BETA'
+    style: 'blue-cumulus'
+  # leave empty if not pertinent
+  homepage_title: 'Tous les défis proposés par data.gouv.fr'
+  homepage_subtitle: 'Retrouvez ici les défis autour de l analyse et de la valorisation de données publiques proposée par l équipe éditorial de data.gouv.fr.'
+  homepage_text_below_hero:
+    display: true
+    html: '<p>Coucou </p><a href="https://www.data.gouv.fr">Data'
+  homepage_text_below_topics:
+    display: true
+    html: '<p>Coucou </p><a href="https://www.data.gouv.fr">toto'
```

Comment:
Suggestions:
- use markdown as in `homepage_subtitle`, keeps coherence — no need for a display attr I think, a `if` should be enough
- ideally refactor homepage settings under a `homepage` parent key